files/fr/web/api/window/frames/index.md

```diff
@@ -26,7 +26,7 @@ frameList = window.frames;
 
 - `frameList` est une liste d'objets frame, semblable à un tableau grâce à la propriété `length` . Ses éléments sont accessibles en utilisant la notation `[i]`.
 - `frameList === window` est évalué à `true`.
-- Chaque élément de `window.frames` est un pseudo tableau représentant l'objet [window](/fr/docs/DOM/window) correspondant au contenu de la  [\<frame>](/fr/docs/HTML/Element/frame) ou [\<iframe>](/fr/docs/HTML/Element/iframe) , et non au DOM element (i)frame (ex., `window.frames[ 0 ]` équivaut à `document.getElementsByTagName( "iframe" )[ 0 ].contentWindow`).
+- Chaque élément de `window.frames` est un pseudo tableau représentant l'objet [window](/fr/docs/DOM/window) correspondant au contenu de la [\<frame>](/fr/docs/HTML/Element/frame) ou [\<iframe>](/fr/docs/HTML/Element/iframe) , et non au DOM element (i)frame (ex., `window.frames[ 0 ]` équivaut à `document.getElementsByTagName( "iframe" )[ 0 ].contentWindow`).
```

Comment:
The ignore was originally from a bug in Markdownlint for `document.getElementsByTagName( "iframe" )[ 0 ].contentWindow`) being parsed as a backwards link. It's fixed in the new version, so this can be dropped. Should have documented it better before